01-js-intro-and-jquery/app.js

```diff
@@ -0,0 +1,27 @@
+var main = function() {
```

Comment:
**Minor:** space after ``function`` definition (it should be: ``function () { // MAGIC! :O }``).